packages/default/scss/grid/_layout.scss

```diff
@@ -376,8 +376,15 @@
             vertical-align: middle;
         }
         .k-button + .k-button {
+            margin-left: 0;
+        }
+        .k-button + .k-button-group .k-button:first-of-type {
+            margin-left: $grid-toolbar-button-spacing;
+        }
+        .k-button-group + .k-button {
```

Comment:
The same way we can have buttons after button groups, we can have button groups after button groups; toolbars after buttons; comboboxes after toolbars etc.